lms/djangoapps/courseware/views.py

```diff
@@ -1577,12 +1594,11 @@ def financial_assistance_form(request):
     enrolled_courses = [
         {'name': enrollment.course_overview.display_name, 'value': unicode(enrollment.course_id)}
         for enrollment in CourseEnrollment.enrollments_for_user(user).order_by('-created')
-        if CourseMode.objects.filter(
+        if enrollment.mode != CourseMode.VERIFIED and CourseMode.objects.filter(
```

Comment:
Ah good point.  Maybe a comment then?  It's just a little bit much to look at and quickly understand what this is doing.